testdata/fuzz/FuzzMultiProofDeserialize/2551f538de792b6d

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
```

Comment:
All these files were the cases that discovered the mentioned inconsistencies or bugs. They're now part of tests automatically when running `go tests` to avoid regressions.